src/components/group-add-post.js

```diff
@@ -52,6 +53,12 @@ class GroupAddPost extends React.Component {
       [name]: value
     })
   }
+
+handleEditorChange = (content) => {
+    this.setState({text: content.level.content})
+	console.log(content.level.content)
```

Comment:
Don't leave `console.log` in the code. This reflects in production environment too.